documentation/specs/TabNav.md

```diff
@@ -0,0 +1,81 @@
+# `TabNav` Component Specification
```

Comment:
since the name `Tabs` more often refers to the W3C Tabs ARIA pattern, which is not meant for navigation, i thought the name `TabNav`, which is what [Github's Primer](https://primer.style/design/components/tab-nav) uses for their version of tabs that navigate, would be more accurate of a name. per usual, i don't necessarily _love_ it. i've seen various different names in different component libraries for this component. open to suggestions

Reply:
Sounds good to me!

Reply:
agree, this is a sensible choice